kentik_api_library/README.md

```diff
@@ -68,6 +68,32 @@ The general approach is that every single KentikAPI resource is represented in t
 - [Tenant](./kentik_api/public/tenant.py)
 - [User](./kentik_api/public/user.py)
 
+## Additional utilities available in the `utils` sub-module
+### Authentication support
+- `get_credentials`: function for retrieving authentication credentials from the environment or a profile stored on disk.
+  API authentication credentials can be provided via environment variables `KTAPI_AUTH_EMAIL` and `KTAPI_AUTH_TOKEN`
+  or via named profile (specified as argument to the `get_credentials` functions, defaulting to `default`) which is
+  a JSON file with following format:
+```
+{
+  "email": "<email address>",
+  "api-key": "<the API key>"
+}
+```
+Path to the profile file can be provided in `KTAPI_CFG_FILE`. Otherwise it is first searched in 
+`${KHOME}/<profile_name>` and then in `${HOME}/.kentik/<profile_name>`.
```

Comment:
Probably: `${HOME}/<profile_name>`

Reply:
No, the intent is to allow additional location provided via `KHOME` (e.g. `/etc/kentik`), in case the user does not want to have the profiles in `$HOME/.kentik`.

Reply:
I suggest renaming to KTHOME to follow the already established naming convention.

Reply:
Fair enough. Will do.